parseComCode/ReadMeParseComCloudCode.txt

```diff
@@ -11,10 +11,36 @@ ParseComTools\parse new
 This asks for an email, which is parse@bloomlibrary.org, and a password, which you will need to know!
 After you enter the right password, it asks which app you want to make cloud code for. Pick an appropriate number from the list.
 This creates a folder under whatever was the current directory when you did it...for example mine is at the root of I:.
+
 The important file (apart from some keys you should protect) is parse/cloud/main.js.
 Replace that file with the version checked in here.
 Edit it however you need to.
+
+The CLI produces a file parse/config/global.json. This file appears to be incompatible with the deploy function.
```

Comment:
CLI = command language interpreter? Do you mean this happens when you run parse new, or every time you deploy, or...? Have you found any indication of whether this is a temporary bug or (for some reason) a new requirement? Might be helpful to say a little more about when and how often you have to make this manual change.